Create compatability.mdx for Consul Kubernetes docs (#8655)
hashicorp/consul: website/pages/docs/k8s/upgrade/compatibility.mdx

---
layout: docs
page_title: Compatibility Matrix
sidebar_title: Compatibility Matrix
description: Compatibility Matrix for Consul Kubernetes and Consul
---

# Compatibility Matrix for Consul Kubernetes and Consul 

> Consul Kubernetes (consul-k8s) is released through Consul Helm. For every release of Consul Kubernetes, a new version of the Consul Kubernetes 

Reviewer: ```suggestion
Consul Kubernetes (consul-k8s) is managed using Consul Helm. For every release of Consul Kubernetes, a new version of the Consul Kubernetes 
```

> Helm chart and Consul Kubernetes binary is released through the HashiCorp Helm repository. The general best practice is to upgrade

Reviewer: ```suggestion
Helm chart and Consul Kubernetes binary is released through the HashiCorp Helm repository. The recommended best practice is to upgrade
```

> the Helm chart for upgrades of Consul Kubernetes since Consul Kubernetes binaries may have dependencies with certain versions of

Reviewer: ```suggestion
the Helm chart which will ensure a compatible version of the Consul Kubernetes binary is used.
```

> the Helm chart. 

Reviewer: ```suggestion
```
